### PR TITLE
Add example showing the OpenTelemetry Collector used to receive Docker logs

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

File: examples/splunk-hec/logging/main.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	counter := int64(0)
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM, syscall.SIGQUIT)
+	ticker := time.NewTicker(1 * time.Second)
+
+	logger.Info("Start logging app")
+	for {
+		select {
+		case <-ticker.C:
+			time.Sleep(1 * time.Second)
```
Comment (rmfitzpatrick) [inline examples/splunk-hec/logging/main.go:26]: Is this still desired when triggered by the ticker?

**Review by rmfitzpatrick — APPROVED** [review]

File: examples/splunk-hec/logging/main.go (hunk 1 above)
Comment (atoulme) [reply examples/splunk-hec/logging/main.go:26]: No, you're right, I can remove that line.

Comment (atoulme) [reply examples/splunk-hec/logging/main.go]: Done.

**Review by flands — APPROVED** [review]
LGTM

**Pull request merged**